package/firmware/wireless-regdb/Makefile

```diff
@@ -20,7 +20,7 @@ define Package/wireless-regdb
 endef
 
 define Build/Compile
-	python $(PKG_BUILD_DIR)/db2fw.py $(PKG_BUILD_DIR)/regulatory.db $(PKG_BUILD_DIR)/db.txt
+	$(PYTHON2) $(PKG_BUILD_DIR)/db2fw.py $(PKG_BUILD_DIR)/regulatory.db $(PKG_BUILD_DIR)/db.txt
```

Comment:
Backporting b6bae4a2c9f11f7e55319c2b4c709396ce649688  would probably make sense here.

Reply:
Either way, this should be just $(PYTHON) not PYTHON2 explicitly, as python2 is implicit in 19.07.

Reply:
https://github.com/openwrt/openwrt/commit/b6bae4a2c9f11f7e55319c2b4c709396ce649688 is an ugly way to fix python2 python3 confusion, we should just stop using python without the version number, it makes everything easier to understand and everything working